cooperator_portal/controllers/portal.py

```diff
@@ -43,17 +43,21 @@ def _prepare_portal_layout_values(self):
             .sudo()
             .search([("partner_id", "in", [coop.id])], limit=1)
         )
-        invoice_mgr = request.env["account.invoice"]
-        capital_request_count = invoice_mgr.search_count(
+        AccountMove = request.env["account.move"]
```

Comment:
it feels strange to me to have MixedCase variable that is not a class (since it is bound to a context). i would prefer something like `account_move_model`.